File: Java/Lector.java
import java.util.Scanner;
public class Lector {
    Scanner sc = new Scanner(System.in);
   public double leerNumero(String mensaje){
    System.out.print(mensaje);
    return(sc.nextDouble());
   }
   public String leerTexto(String mensaje){
    System.out.print(mensaje);
    return (sc.next());
   }
}

File: C#/Lector.cs
using System;
public class Lector
{
    public double LeerNumero(string mensaje)
    {
        Console.Write(mensaje);
        return Convert.ToDouble(Console.ReadLine());
    }
    public string LeerTexto(string mensaje)
    {
          Console.Write(mensaje);
          return Console.ReadLine();
    }
  
}